e2e/jenkins/crw-ci/k8s/pr-check/Jenkinsfile

```diff
@@ -0,0 +1,229 @@
+        /usr/local/bin/minikube delete 
+    """
+                        }
+                    }
+                }
+
+                stage("Get sources") {
+                    steps {
+                        checkout([$class                           : 'GitSCM',
+                                  branches                         : [[name: "*/${ghprbSourceBranch}"]],
+                                  doGenerateSubmoduleConfigurations: false,
+                                  extensions                       : [[$class           : 'RelativeTargetDirectory',
+                                                                       relativeTargetDir: 'che']],
+                                  userRemoteConfigs                : [[url: "https://github.com/${ghprbGhRepository}.git"]]])
+                    }
+                }
+            }
+        }
+
+        stage("Prepare to start Che on K8S") {
+            failFast true
+
+            parallel {
+                stage("Download chectl") {
+                    steps {
+                        script {
+                            sh """
+  wget --no-verbose https://github.com/che-incubator/chectl/releases/latest/download/chectl-linux
+  sudo mv chectl-linux /usr/local/bin/chectl
+  sudo chmod +x /usr/local/bin/chectl
+"""
+                        }
+                    }
+                }
+
+                stage("Start Kubernetes") {
+                    steps {
+                        script {
+                            sh """
+  /usr/local/bin/minikube start \\
+    --vm-driver=none \\
+    --cpus 4 \\
+    --memory 12288 \\
+    --logtostderr
+
+  sudo chmod g+r -R /home/codenvy/.minikube
+  sudo chmod g+r -R /home/codenvy/.kube
+  sudo chmod g+r /var/lib/kubeadm.yaml
+"""
+                        }
+                    }
+                }
+
+                stage("Build Che") {
+                    steps {
+                        script {
+                            sh "mvn clean install -f ${WORKSPACE}/che/pom.xml"
+                        }
+                    }
+                }
+            }
+        }
+
+        stage("Push Che artifacts") {
+            steps {
+                withCredentials([string(credentialsId: 'ed71c034-60bc-4fb1-bfdf-9570209076b5', variable: 'maxura_docker_password')]) {
+                    script {
+                        sh """
+  ${WORKSPACE}/che/dockerfiles/che/build.sh --organization:maxura --tag:${CHE_IMAGE_TAG} --dockerfile:Dockerfile
+  docker tag ${CHE_IMAGE_REPO}:${CHE_IMAGE_TAG} docker.io/${CHE_IMAGE_REPO}:${CHE_IMAGE_TAG}
+  docker login -u maxura -p ${maxura_docker_password}
+  docker push docker.io/${CHE_IMAGE_REPO}:${CHE_IMAGE_TAG}
+"""
+                    }
+
+                }
+            }
+        }
+
+        stage("Start Che") {
+            steps {
+                script {
+                    sh """
+  /usr/local/bin/chectl server:start \\
+   --k8spodreadytimeout=180000 \\
+   -t=${WORKSPACE}/deploy/ \\
```

Comment:
Where did this `${WORKSPACE}/deploy/` folder came from? eclipse/che git repo is cloned into relative dir `che`... Shouldn't this be `${WORKSPACE}/che/deploy`?

Reply:
Good catch!
There is `che` project content which had been checked out at one of the previous build of the job on `codenvy-slave9`.